www/index.html

```diff
@@ -85,26 +100,67 @@
             left: 2%;
         }
 
+        .containerCustom {
+            @apply(--layout-flex);
+            @apply(--layout-vertical);
+            width: 100%;
+            height: 100%;
+        }
+
+        paper-tabs {
+
+            --paper-tabs-selection-bar-color: var(--paper-green-700);
+            --paper-tabs-selection-bar: {
+                height: 4px;
+            }
+        }
+
+        .flex-center-justified {
+            @apply(--layout-horizontal);
+            @apply(--layout-center-justified);
+            @apply(--layout-vertical-center);
+        }
+
+        .cafe-header {
+            @apply(--paper-font-headline);
+        }
+
+        .cafe-light {
+            color: var(--paper-grey-600);
+        }
+
+        .cafe-location {
+            float: right;
+            font-size: 15px;
+            vertical-align: middle;
+        }
+
+        .cafe-reserve {
+            color: var(--google-blue-500);
+        }
+
+        iron-icon.star {
+            --iron-icon-width: 16px;
+            --iron-icon-height: 16px;
+            color: var(--paper-amber-500);
+        }
+
+        iron-icon.star:last-of-type {
+            color: var(--paper-grey-500);
+        }
+
     </style>
+
 </head>
 
-<body>
+<body unresolved>
```

Comment:
Is FOUC happening on all the whole app? If doesn't, I think you should add this attribute only on elements that are slow to load correctly.
Because for UX reasons, I think it's bad to show a totally blank app, we should see a few elements loaded.